scheduler/filter/expr.go

```diff
@@ -38,7 +38,7 @@ func parseExprs(key string, env []string) ([]expr, error) {
 
 					// validate key
 					// allow alpha-numeric
-					matched, err := regexp.MatchString(`^(?i)[a-z_][a-z0-9\-_]+$`, parts[0])
+					matched, err := regexp.MatchString(`^(?i)[a-z_][a-z0-9\-_.]+$`, parts[0])
```

Comment:
A tiny bit unsure about the effect on the regex: `.` is supposed to mean *any character*.

Is this being interpreted as a dot or as any character and do we need to escape it?